transforms/language/lang_id/python/pyproject.toml

```diff
@@ -12,7 +12,7 @@ dependencies = [
     "data-prep-toolkit==0.2.1.dev0",
     "fasttext==0.9.2",
     "langcodes==3.3.0",
-    "huggingface-hub==0.21.4",
+    "huggingface-hub >= 0.21.4, <1.0.0",
```

Comment:
Note: this is needed to avoid conflicts between transforms and other dependencies.